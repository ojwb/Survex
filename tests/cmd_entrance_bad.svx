; pos=fail warn=1 error=1
*fix 1 reference 1 0 0
*entrance
*entrance 2
