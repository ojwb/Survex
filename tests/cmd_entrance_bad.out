./cmd_entrance_bad.svx:3:10: error: Expecting station name
 *entrance
          ^

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
./cmd_entrance_bad.svx:4: warning: Station "2" referred to by *entrance or *export but never used
   1 0-node.
There were 1 warning(s) and 1 error(s) - no output files produced.
